### PR TITLE
Gdbn

### PR DESCRIPTION
**Pull request by nick-youngblut**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by nick-youngblut** [comment]
@conda-forge/python Why isn't [gnumpy](https://github.com/conda-forge/gnumpy-feedstock) available on conda-forge yet? It was merged on Oct 3.

**Comment by marcelotrevisani** [comment]
> @conda-forge/python Why isn't [gnumpy](https://github.com/conda-forge/gnumpy-feedstock) available on conda-forge yet? It was merged on Oct 3.

For some reason, the package was not created. So, I opened this pull request https://github.com/conda-forge/gnumpy-feedstock/pull/1 to recreate the package. After the CI passes could you please merge it?

**Comment by nick-youngblut** [comment]
@marcelotrevisani Any idea on what's going on with the win build error:

```
Error: [('C:\\bld\\gdbn_1570388643594\\work\\local\\license.txt', 'c:\\users\\appveyor\\appdata\\local\\temp\\1\\pip-req-build-r4p3eo\\local\\license.txt', "[Errno 22] invalid mode ('rb') or filename: 'C:\\\\bld\\\\gdbn_1570388643594\\\\work\\\\local\\\\license.txt'"), ('C:\\bld\\gdbn_1570388643594\\work\\local\\mnistExample.py', 'c:\\users\\appveyor\\appdata\\local\\temp\\1\\pip-req-build-r4p3eo\\local\\mnistExample.py', "[Errno 22] invalid mode ('rb') or filename: 'C:\\\\bld\\\\gdbn_1570388643594\\\\work\\\\local\\\\mnistExample.py'"), ('C:\\bld\\gdbn_1570388643594\\work\\local\\README.txt', 'c:\\users\\appveyor\\appdata\\local\\temp\\1\\pip-req-build-r4p3eo\\local\\README.txt', "[Errno 22] invalid mode ('rb') or filename: 'C:\\\\bld\\\\gdbn_1570388643594\\\\work\\\\local\\\\README.txt'")]
```

Something wrong with the license file or how it's set in the meta.yaml?

File: recipes/gdbn/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (marcelotrevisani) [inline recipes/gdbn/meta.yaml]: could it be ``noarch``?
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
  noarch: python
```

Comment (nick-youngblut) [reply recipes/gdbn/meta.yaml]: It appears that `noarch: python` did not help. I'm getting the error: 

```
(%PREFIX%) %SRC_DIR%>%PREFIX%\python.exe -m pip install . --no-deps -vv 
Ignoring indexes: https://pypi.org/simple
Created temporary directory: c:\users\vssadm~1\appdata\local\temp\pip-ephem-wheel-cache-xiiw0r
Created temporary directory: c:\users\vssadm~1\appdata\local\temp\pip-req-tracker-miij7h
Created requirements tracker 'c:\\users\\vssadm~1\\appdata\\local\\temp\\pip-req-tracker-miij7h'
Created temporary directory: c:\users\vssadm~1\appdata\local\temp\pip-install-ixww6o
Config variable 'Py_DEBUG' is unset, Python ABI tag may be incorrect
Config variable 'WITH_PYMALLOC' is unset, Python ABI tag may be incorrect
Config variable 'Py_UNICODE_SIZE' is unset, Python ABI tag may be incorrect
Processing c:\miniconda\conda-bld\gdbn_1572808658578\work
  Created temporary directory: c:\users\vssadm~1\appdata\local\temp\pip-req-build-oi8tcv
Cleaning up...
Removed build tracker 'c:\\users\\vssadm~1\\appdata\\local\\temp\\pip-req-tracker-miij7h'
Traceback (most recent call last):
  File ".ci_support\build_all.py", line 134, in <module>
    build_all(args.recipes_dir, args.arch)
  File ".ci_support\build_all.py", line 68, in build_all
    build_folders(recipes_dir, new_comp_folders, arch, channel_urls)
  File ".ci_support\build_all.py", line 124, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "C:\Miniconda\lib\site-packages\conda_build\api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "C:\Miniconda\lib\site-packages\conda_build\build.py", line 2344, in build_tree
    notest=notest,
  File "C:\Miniconda\lib\site-packages\conda_build\build.py", line 1459, in build
    windows.build(m, build_file, stats=build_stats, provision_only=provision_only)
  File "C:\Miniconda\lib\site-packages\conda_build\windows.py", line 294, in build
    check_call_env(cmd, cwd=m.config.work_dir, stats=stats, rewrite_stdout_env=rewrite_env)
  File "C:\Miniconda\lib\site-packages\conda_build\utils.py", line 398, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "C:\Miniconda\lib\site-packages\conda_build\utils.py", line 378, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['cmd.exe', '/d', '/c', 'conda_build.bat']' returned non-zero exit status 1.
```

**Pull request closed without merging**